library_generation/owlbot/templates/java_library/.github/workflows/update_generation_config.yaml

```diff
@@ -18,7 +18,7 @@ on:
   schedule:
   - cron: '0 2 * * *'
   workflow_dispatch:
-
+{% raw %}
```

Comment:
We don't want to render `${{ secrets.CLOUD_JAVA_BOT_TOKEN }}`, use `raw` to ignore the variable.